clang/lib/Sema/SemaChecking.cpp

```diff
@@ -2156,6 +2156,24 @@ static ExprResult PointerAuthAuthAndResign(Sema &S, CallExpr *Call) {
   return Call;
 }
 
+static ExprResult PointerAuthStringDiscriminator(Sema &S, CallExpr *call) {
```

Comment:
Naming style of variables all over the function (use `PascalCase` https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly)

Reply:
Clang originally had different conventions;  it usually gets (has gotten) updated over time for older code, I suppose now's the time for these changes.